helpdesk/views/public.py

```diff
@@ -77,6 +77,10 @@ def homepage(request):
 def view_ticket(request):
     ticket_req = request.GET.get('ticket', '')
     email = request.GET.get('email', '')
+    # If there is no email address in the query string, get it from
+    # the currently logged-in user
+    if email == '':
+        email = request.user.email
```

Comment:
Yes. Better would be a redirect to a login page. I think I'd like to punt on this for now, (since we have two cards for doing the CMS integration and for fixing several similar issues related to permissions). I think one or both of those cards will end up addressing this.